docs/getting-started.ts

```diff
@@ -36,19 +76,32 @@ async function main(): Promise<void> {
     },
     type: 'object',
   })
-
   /* To store the CTYPE on the blockchain, you have to call: */
-  const identity = Kilt.Identity.buildFromMnemonic(
-    'receive clutch item involve chaos clutch furnace arrest claw isolate okay together',
-    // using ed25519 as key type because this is how the endowed identity is set up
-    { signingKeyPairType: 'ed25519' }
+  const accountMnemonic =
+    'receive clutch item involve chaos clutch furnace arrest claw isolate okay together'
+  const account = keyring.addFromMnemonic(accountMnemonic)
+
+  const onChainDid = await Kilt.Did.createOnChainDidFromSeed(
```

Comment:
I would perhaps rename it to `fullDid`? I know the method says `createOnChainDidFromSeed` but using a different name might be confusing.